packages/shared-core/src/sdk/documents/users.ts

```diff
@@ -70,7 +70,7 @@ export function hasAppCreatorPermissions(user?: User | ContextUser): boolean {
   return _.flow(
     _.get("roles"),
     _.values,
-    _.find(x => x === "CREATOR"),
+    _.find(x => ["CREATOR", "ADMIN"].includes(x)),
```

Comment:
Admin roles was inferred from other places of the model, but I've added this in addition, just in case.

Reply:
Is this "CREATOR" (and now "ADMIN") not typed in some enum we can use, instead of having magic strings?

Reply:
That roles are defined in the fronted module, I imagine we don't have to create dependencies from fronted to shared. Maybe we need to move that constant here. Maybe in a separated issue? What do you think?

Reply:
https://linear.app/budibase/issue/GROW-294/move-role-enum-to-shared-core